extensions/windows-event-log/wel/JSONUtils.cpp

```diff
@@ -162,9 +186,4 @@ std::string jsonToString(const rapidjson::Document& doc) {
   doc.Accept(writer);
   return buffer.GetString();
 }
-
-}  // namespace wel
-}  // namespace minifi
-}  // namespace nifi
-}  // namespace apache
-}  // namespace org
+} // namespace org::apache::nifi::minifi::wel
```

Comment:
I think the linter failure comes from here `JSONUtils.cpp:189:  At least two spaces is best between code and comments  [whitespace/comments] [2]`

Reply:
thanks, fixed